src/20_Components/amp-bind.html

```diff
@@ -79,6 +88,25 @@
 
   `[property]="expression"`
 
+-->
+
+<!-- #### Text for textContent -->
+<!--
+The text of this &lt;p&gt; will change when the `currentAnimal` variable
```

Comment:
If you use the exact same wording, that you should copy the whole thing. Add 'either the `allAnimals` or '.